packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -352,5 +381,6 @@ const target = new chatbot.SlackChannelConfiguration(stack, 'MySlackChannel', {
   slackChannelId: 'YOUR_SLACK_CHANNEL_ID',
 });
 
-const rule = pipeline.notifyOnExecutionStateChange('NotifyOnExecutionStateChange', target);
+const rule = pipeline.notifyOnExecutionStateChange('NotifyOnExecutionStateChange'
+, target);
```

Comment:
Like I mentioned already in https://github.com/aws/aws-cdk/pull/16468#discussion_r708538948, can we please revert all of these formatting changes to this file, and leave only the new paragraph introducing `enableKeyRotation`?